### PR TITLE
make changes for the january 10th version of this thing

### PR DESCRIPTION
**Pull request by nickgartmann**
- [x] Get rid of references to PKWARE
- [x] Change addresses
- [x] Change dates
- [x] Change times

@mitchellhenke @joshdholtz @stoodder @jakerobers @civilframe

### CODE REVIEW

**Comment by jeregrine** [comment]
:+1:

File: index.html [hunk 1]
```diff
@@ -28,25 +28,21 @@
...
       <h3>
-        An 8 hour event<br>
+        A 10 hour event<br>
         Build something and eat free food
       </h3>
       <h5 style="font-weight: 100;">
-        Oct 25th, 2014 10:00am - 6:00pm<br>
-        201 E Pittsburgh Ave, Suite 400, Milwaukee, WI 53204
+        Jan 10th, 2015 10:00am - 8:00pm<br>
+        170 S 1st St, Milwaukee, WI 53204
```
Comment (mitchellhenke) [inline index.html:42]: 170 S 1st St, or 107 S 1st St?

Comment (nickgartmann) [inline index.html:42]: Idk, whichever is correct?

File: index.html [hunk 2]
```diff
@@ -72,18 +68,17 @@ <h5 style="font-weight: 100;">
...
     <h2>When is it?</h2>
-    <p class="center">October 25th, 2014<br> 10:00am - 6:00pm</p>
+    <p class="center">January 10th, 2015<br> 10:00am - 8:00pm</p>
     <h2>Where is it?</h2>
     <p class="center">
       <em>
-        PKWARE Offices<br>
-        201 E Pittsburgh Ave, Suite 400<br>
+        The Commons<br>
+        107 S 1st St.<br>
```
Comment (mitchellhenke) [inline index.html]: This says 107, the one above says 170

Comment (nickgartmann) [inline index.html]: OH! Thats the difference between the two above. couldn't see it, thought it was the comma. will confirm

File: index.html [hunk 3]
```diff
@@ -72,18 +68,17 @@ <h5 style="font-weight: 100;">
       All we want is to spend time with other people who want to <em>create</em>.
     </p>
     <h2>When is it?</h2>
-    <p class="center">October 25th, 2014<br> 10:00am - 6:00pm</p>
+    <p class="center">January 10th, 2015<br> 10:00am - 8:00pm</p>
     <h2>Where is it?</h2>
```
Comment (mitchellhenke) [inline index.html:72]: Do we need more specific instructions on getting there?  I've never been, and it just looks like Colectivo

Comment (nickgartmann) [inline index.html:72]: Fair, i'll ask cordio about how he normally tells people how to get up there

**Comment by alexnsolo** [comment]
:+1:

**Pull request merged**